README.md

```diff
@@ -27,13 +26,6 @@ nvm install
 nvm use
 ```
 
-Ensure that the aws cli is configured and that the default output format is either JSON or None:
```

Comment:
If we are using the AWS CLI in a script somewhere and care about the format of the output, we should be using the `--json` flag in that script.